config/common/common-packages.yml

```diff
@@ -1,13 +1,15 @@
 type: rpm-ostree
 repos: 
   - https://copr.fedorainfracloud.org/coprs/secureblue/hardened_malloc/repo/fedora-39/secureblue-hardened_malloc-fedora-39.repo
+  - https://copr.fedorainfracloud.org/coprs/34n0s/pam-authramp/repo/fedora-39/pam-authramp.repo
```

Comment:
for sake of consistency when users refresh their repos, let's keep all the coprs for secureblue in the secureblue org account: https://copr.fedorainfracloud.org/coprs/secureblue/pam-authramp/builds/

this will keep everything centralized and "secureblue branded" and also give secureblue the flexibility of delaying new versions if need be compared to your official copr.